### DISCUSSION TITLE
How to make a full Backup of all Website created with PUBLII on Windows 10 ?

### DISCUSSION THREAD

**Discussion by hadyman** [GetPublii/Publii#1845] [General]
Hello I bought a new computer (-_-) and i need to know how to make a full Backup of all Website (>25) that I created with PUBLII on my old computer (Windows 10) and then move it to my new computer (Windows 11) ?
Thanks for your help

**Comment by bobmitro** [comment]
Copy the "sites" folder from this location: `/Users/USER_NAME/Documents/Publii` to the new device.
You can also consider copying other folders from the same location, such as plugins, themes, logs, etc.

**Comment by hadyman** [comment]
Hi Bob thank you !
This what I did but few problems :

1. 90% of my posts are missing :(
2. when i clic on pages tab i got this alert https://disk.yandex.com/i/N3yU8ca6QPq0kg

thanks

**Reply by bobmitro** [reply]
> 90% of my posts are missing :(

It's not possible. 

The "pages note" just inform you that your theme does not support pages. So update your theme.

**Reply by hadyman** [reply]
bob when i log to your website and try to download my purchase plugins or template i got error 503 see here https://disk.yandex.com/i/VoGplxc4jBo2Yg

**Comment by michaelfluegge** [comment]
I did it the way Bob described it, and it worked pretty well. 
Consider reading the documentation. It is always a good idea: https://getpublii.com/docs/website-backups.html

**Reply by hadyman** [reply]
Hi yes i read the documentatiion before coming here to ask question but somehow PUBLII show me this notification https://disk.yandex.com/i/XxABsxIsEQKIrw
this is the old PATH from the oldest Disk
